src/utils/common.ts

```diff
@@ -78,6 +78,16 @@ export function max(a: BigNumberish, b: BigNumberish): BN {
   return bna.gte(bnb) ? bna : bnb;
 }
 
+/**
+ * Compute the absolute value of a BigNumber
+ * @param _x The BigNumber to compute the absolute value of
+ * @returns The absolute value of the given BigNumber
+ */
+export function abs(_x: BigNumberish): BN {
+  const x = toBN(_x);
```

Comment:
Can't you just `return x.abs()`?

Reply:
Yep! Resolved. https://github.com/across-protocol/sdk-v2/pull/236/commits/4ea323da458aab0851327499ce6276e83674aa49